### PR TITLE
move lesson and update with scope rules and shadowing example

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,193 @@
...
+main block scope:  2 [] 1 
+anotherMethod block scope: 1 5 
+```
+
+## Shadowing
+
+- A *simple* name is a single identifier, such as `age` or `ssn`.
+- A *qualified* name consists of a name, a `.`, and an identifier, such as `this.age` or `employee1.ssn`.
+
+Shadowing happens a variable declared in one scope (the nested scope)
```
Comment (kcleland0818) [inline README.md]: Typos: "Shadowing happens a variable declared in one scope (the nested scope) has the same name as a variable declared in the enclosing scope."

I believe this should be "Shadowing happens when a variable is declared in one scope (the nested scope) and has the same name as a variable declared in the enclosing scope."

File: README.md [hunk 2]
```diff
@@ -1,32 +1,193 @@
...
+has the same name as a variable declared in the enclosing scope.
+In this case, the declaration in the nested scope
```
Comment (kcleland0818) [inline README.md]: Maybe move this sentence (and the one after it) after showing the code example?

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: yes, fixed.

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: Moved and further explained.

**Review by kcleland0818 — APPROVED** [review]

**Review by developerjaym — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**